nucleus/parent/pom.xml

```diff
@@ -79,7 +79,7 @@
         <findbugs.version>3.0.3</findbugs.version>
         <findbugs.glassfish.logging.validLoggerPrefixes>javax.enterprise</findbugs.glassfish.logging.validLoggerPrefixes>
 
-        <webservices.version>2.4.3</webservices.version>
+        <webservices.version>3.0.0-M1</webservices.version>
```

Comment:
just for the record - this brings in dependencies NOT yet approved by Eclipse EMO, therefore also NOT available in maven central yet. They won't get to central before the approval from EMO will be given.

Reply:
yeah we are just trying to get GlassFish 6.0.0 RCs built so not too bothered by EMO approval at the moment as long it is in jakarta staging.

Reply:
There has been some confusion yes, this is indeed all in staging only. 

Since Metro is not an API project (and I think doesn't use the spec plug-in, but I double check), I can use another suffix as well instead of -M1 if that makes things more clear?

Although it hasn't been set at the platform level I kinda assumed that -RC1 would be used for the Jakarta-ized version, but maybe I'm wrong there.

Reply:
there were many discussions and I believe the conclusion was use whatever suffix project wants but RC is recommended for APIs and M for impls. Ie hibernate artifacts are using ".Alpha" suffix to give different example

Reply:
Feedback addressed by re-staging as `-alpha1`. Thanks for your valuable feedback @lukasj